apps/aecore/test/aec_conductor_tests.erl

```diff
@@ -99,6 +113,63 @@ test_run_miner() ->
     ?assertEqual(ok, ?TEST_MODULE:stop_mining()),
     ok.
 
+miner_timeout_test_() ->
+    {foreach,
+     fun() ->
+             ok = meck:new(aec_pow_cuckoo, [passthrough]),
+             ok = meck:new(application, [unstick, passthrough]),
+             ok = meck:expect(application, get_env, 3,
+                              fun
+                                  (aecore, mining_attempt_timeout, _) ->
+                                      500;
```

Comment:
I am briefly checking if this timeout can be reduced easily (see TODO comment below).

Reply:
The only solution that came to me is:
* Set here a small timeout like 10;
* In the test, mock [`aec_conductor:handle_info(DOWN, ...)`](https://github.com/aeternity/epoch/blob/b6116c807ff746ffd5c1419463800e5ae071b76c/apps/aecore/src/aec_conductor.erl#L340) to send a message to the test pid.

This would show that worker pid died (because killed by timer:exit_after).